activerecord/test/cases/unsafe_raw_sql_test.rb

```diff
@@ -0,0 +1,211 @@
+require "cases/helper"
+require "models/post"
+require "models/comment"
+
+class UnsafeRawSqlTest < ActiveRecord::TestCase
+  fixtures :posts, :comments
+
+  test "order: allows string column name" do
+    enabled, disabled = with_configs(:enabled, :disabled) do
```

Comment:
This test is confusing, what are the possible values for `enabled` and `disabled`? I think in this case is better to duplicate the code than making the test hard to read.